ISSUE_TEMPLATE.md

```diff
@@ -19,5 +19,16 @@ Thanks for opening an issue! A few things to keep in mind:
 <!-- What actually happens? -->
 
 ### How to reproduce
+<!-- 
 
-<!-- For bugs, provide sample code or a repo URL that demos the problem -->
+Your best chance of getting the bug fixed quickly is to provide a REPOSITORY that can be cloned and run.
+
+If you provide a URL, please list the commands required to clone/setup/run your repo e.g.
+
+  $ git clone $YOUR_URL -b $BRANCH
+  $ npm install
+  $ npm start || electron .
+
+You can fork https://github.com/electron/electron-quick-start and submit the branch with your changes
```

Comment:
I would change this wording a bit, so folks don't think we're asking for a pull request or something.

`submit the branch` → `include a link to a branch`